### safety limits on angular positions of joints ###

joint_limits:
  # safety factor
  safety: [0.95]

  # shoulder joints
  left_shoulder:
    
    # limits on shoulder joint degrees of freedom
    pitch: [-2.0857, 2.0857]
    roll: [-0.3142, 1.3265]

